content/applications/general.rst

```diff
@@ -16,130 +16,70 @@ General
-    installations.
-
-.. note:: If you have questions about the content of this section or if you
-    encounter an issue while carrying out these procedures, please contact
-    us through our `support form <https://www.odoo.com/help>`__.
-
-
-Deactivating Users
-------------------
-
-.. note:: Make sure you have sufficient **administrative rights** if
-    you want to change the status of any of your users.
-
-In your Odoo instance, click on **Settings**. You will have a section
-showing you the active users on your database. Click on **Manage Users.**
-
-+----------------+----------------+
-|   |settings|   | |browse_users| |
-+----------------+----------------+
-
-You'll then see the list of your users.
-
-.. image:: general/list_users.png
-    :align: center
-
-.. note:: The pre-selected filter *Internal Users* shows your paying
-    users (different from the *Portal Users* which are free). If you
-    remove this filter, you'll get all your users (the ones you pay for
-    and the portal ones)
-
-In your list of users, click on the user you want to deactivate. As soon
-as you are on the userform, click on the Action drop down menu, and
-then click on Archive.
-
-.. image:: general/deactivate_user.png
-    :align: center
-
-
-The user is now deactivated.
-
-.. danger:: **Never** deactivate the main user (*admin*)
-
-Uninstalling Apps
------------------
-
-.. warning:: Make sure you first test what you are about to do on a
-    :ref:`duplicate <duplicate_online>` of your database before making any
-    changes (*especially* installing/uninstalling apps).
-
-In your Odoo instance click on **Settings**; in this app, you will be
-able to see how many applications you have installed. Click on **Browse
-Apps** to access the list of your installed applications.
-
-+----------------+----------------+
-|   |settings|   | |browse_apps|  |
-+----------------+----------------+
-
-In your applications' dashboard, you will see all the icons of your
-applications. Click on the application you want to uninstall. Then, on
-the form of the application, click on **Uninstall**.
+**Odoo** is a suite of open source business apps that cover all your company needs: :doc:`CRM
+<sales/crm>`, :doc:`eCommerce <websites/ecommerce>`, :doc:`accounting <finance/accounting>`,
+:doc:`inventory <inventory_and_mrp/inventory>`, :doc:`point of sale <sales/point_of_sale>`,
+:doc:`project management <services/project>`, etc. Its unique value proposition is to be at the same
+time *very easy* to use and *fully integrated*.
+
+This section of the documentation covers all functional aspects that aren't app-specific. You can
+find, for example, information about:
+
+- :doc:`Users settings <general/users>` and their :doc:`access rights <general/users/access_rights>`
+- :doc:`Export and import data <general/export_import_data>`
+- :doc:`Payment Acquirers <general/payment_acquirers>` allow your customers to pay you with their
+  favorite payment method.
```

Comment:
```suggestion
- :doc:`Payment Acquirers <general/payment_acquirers>`, to allow your customers to pay you with their
  favorite payment method
```
If you read it the full sentence: "_You can find, for example, information about: Payment Acquirers allow your customers to pay you with their favorite payment method._", it feels weird. With the added comma and "to", it flows better. I would also remove the full stop at the end of that bullet point to be consistent with other bullet points.

Reply:
then, I add full stops at the end of each bullet point, that's more correct ^^